corehq/util/test_utils.py

```diff
@@ -714,3 +715,13 @@ def print_sql_chronologically(self, with_traceback=False, width=150):
             print('\n{}'.format(indent('\n'.join(wrap(out, width)), '\t')))
             if with_traceback:
                 print('\n{}'.format(indent(''.join(query['traceback']), '\t\t')))
+
+
+def db_tests_only(fn):
+    """Skip setUpModule and tearDownModule in non-DB contexts"""
```

Comment:
nit: This comment indicates the context in which this method is used, just a bit confusing.

Reply:
I intended that to be a description of what this does, got any suggestions on how to word it better?